app/classifier/classification-summary.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import tasks from './tasks/index.coffee';
+import apiClient from 'panoptes-client/lib/api-client';
+import { TextSplit } from 'seven-ten';
+
+const ClassificationSummary = (props) => {
+  let firstTimeClassified;
+  if (props.classificationCount === 0) {
+    firstTimeClassified = (
+      <TextSplit
+        splitKey="subject.first-to-classify"
+        textKey="message"
+        splits={props.splits}
+        default={''}
+        elementType={"p"}
+      />
+    );
+  }
+  let body = 'No annotations';
+  if ((props.classification) && (props.classification.annotations.length > 0)) {
+    body = [];
+    for (const annotation of props.classification.annotations) {
```

Comment:
Instead of using a for loop, you could use map which returns an array and assign body to it, `body = props.classification.annotations.map((annotation) => { return (annotation.thing) });`